mathematica/mathematica_body.hpp

```diff
@@ -96,7 +96,7 @@ template<typename D>
 std::string ToMathematica(Quantity<D> const& quantity) {
   std::string s = DebugString(quantity);
   s.replace(s.find("e"), 1, "*^");
-  std::string const number = ToMathematica(quantity / SIUnit<Quantity<D>>());
+  std::string const number = ToMathematica(quantity / quantities::SIUnit<Quantity<D>>());
```

Comment:
We tend to avoid qualified names like `quantities::SIUnit`.  Please put a `using` after line 12 (in namespace principia but not in the subnamespace mathematica).